scripts/gas-refund-program/staking/2.0/StakeV2Resolver.ts

```
@@ -104,7 +111,7 @@ export class StakeV2Resolver extends AbstractStateTracker {
 
     const pspInSePSP2 = sePSP2Balance // 1 BPT = 1 sePSP2
       .multipliedBy(bptPSPBalance)
-      .dividedBy(bptTotalSupply)
+      .dividedBy(bptTotalSupply.isZero() ? new BigNumber(1) : bptTotalSupply)
```

pointless

---

it's better that those stuff blow than let fail silently and bias the data